hal/riscv64/hal.c

```diff
@@ -22,18 +22,19 @@
 #include "../pmap.h"
 #include "../timer.h"
 #include "config.h"
-
+#include "halsyspage.h"
 
 struct {
 	int started;
 } hal_common;
 
-hal_syspage_t *syspage;
 
+syspage_t *syspage;
+addr_t relOffs;
```

Comment:
It's exported to the global namespace, use prefix. `hal_relOffs` or `hal_syspageRelOffs`

Reply:
Thx, done